CHANGELOG.rst

```diff
@@ -5,32 +5,39 @@ Wijzigingen
 1.3.0-rc2 (2022-07-19)
 ===========
 
+New features
+------------
+
+* Add ``indicatieLaatstGezetteStatus`` query parameter to ``Status``
+
 API spec changes
 ----------------
 
-* Removed gerelateerdeExterneZaken from the Zaken resource.
-* Query Statussen on a specific Zaak,  Statustype and indicatieActueleStatus to retrieve the current status.
+* Removed ``gerelateerdeExterneZaken`` from the ``Zaak`` resource.
 
 1.3.0-rc1 (2022-07-06)
 ===========
 
+New features
+------------
+
+* Add missing RGBZ attributes for existing resources. Among other things to supprt TMLO.
+* Added validation of the ``Zaak`` resource with a `gearchiveerd` status to 
+  related resources.
+* Ordering of a list of ``Zaken`` can be done on the following attributes:
+  - ``startdatum``
+  - ``startdatum``
+  - ``einddatum``
+  - ``-einddatum``
+  - ``publicatiedatum``
+  - ``-publicatiedatum``
+  - ``archiefactiedatum``
+
 API spec changes
 ----------------
 
-* Reverted expanded relations
```

Comment:
De `expand` query parameter is nooit onderdeel geweest van een release of RC (en hoeft volgens mij daarom ook niet genoemd te worden)